test/config.ts

```diff
@@ -1,9 +1,16 @@
-import {argv} from "process";
-import {Log} from "../src/log";
+import { argv } from "process";
+import { Log } from "../src/log";
+import * as WhyRunning from "why-is-node-running";
+
+const logger = new Log("MessageProcessor");
 
 // we are a test file and thus need those
 /* tslint:disable:no-unused-expression max-file-line-count */
 
 if (!argv.includes("--noisy")) {
     Log.ForceSilent();
 }
+
+after(() => {
```

Comment:
Wait `after(()=>)` waits for the tests to all finish and then run things, that's pretty cool :o. Would this fire every time? I'm wondering if we should have some accompanying logging.

Reply:
it fires once after all tests are done!